internal/commands/test.go

```diff
@@ -155,20 +155,22 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("get configurations: %w", err)
 			}
 
-			policyPath := viper.GetString("policy")
+			policyPaths := viper.GetStringSlice("policy")
 			urls := viper.GetStringSlice("update")
-			for _, url := range urls {
-				sourcedURL, err := policy.Detect(url, policyPath)
-				if err != nil {
-					return fmt.Errorf("detect policies: %w", err)
-				}
+			for _, policyPath := range policyPaths {
```

Comment:
This will now download all the URLs in all policy directories, which means policies are duplicated across all policy directories. Maybe we can circumvent this by creating a second policies flag (e.g. `--policy-layers`)

Reply:
Oops, that was a bit sloppy reading of the existing code before looping over it. 
I haven't used this feature before, so not sure what would be the best ergonomics, but given support for multiple directories, perhaps it makes sense to keep the downloaded policies in a distinct directory, maybe even something like `remote-policies/<name>/` or similar?

Reply:
The download functionality was added for CI/CD pipelines to support a one liner that can both fetch the policies and execute the tests in the same command. If we where to move the policies by default in a remote-policies directory that would lead to breaking existing pipelines and lead to non-obvious behaviour (all of the sudden you need to also specify the `--policy` flag and pass in the `remote-policies` dir). I think we need to find an alternate solution.

Reply:
It might not break CI oneliners if we'd automagically add the `--update` targets to the policy directory list, but it'd break multistep workflows where you'd have previously downloaded separately, right?
The simple way to not break existing usage would be to just download to the first policy path. Then it'll keep working just like the current state. But might not be very obvious still.

Reply:
I guess using the first directory as the download directory is not the worst. The most common use would have be to have a shared company policy + additional local policies. In that case downloading the shared policies in the first specified policy dir would make sense. Slice orders should be consistent so using the first flag should work.

It still feels weird to rely on the order of flags passed. But I guess it is the best option so far.

Reply:
I can't really come up with any other idea which is backwards compatible, I'm afraid. Shall we consider that "good enough", or can we come up with something better?

Reply:
Good enough sounds like the way to go, I can't come up with a better solution either. So let's go for the solution where we download the policies specified in the `--update` flag to the first policy path specified and make sure the behaviour is documented.